hydromt_fiat/fiat.py

```diff
@@ -1284,23 +1284,40 @@ def read_geoms(self):
             ]
             self.exposure.read_geoms(exposure_fn)
 
+        
         fns = glob.glob(Path(self.root, "geoms", "*.geojson").as_posix())
         if self.spatial_joins["aggregation_areas"]:
-            fns_aggregation = glob.glob(Path(self.root, "geoms", "aggregation_areas","*.geojson").as_posix())
-            fns.append(fns_aggregation[0])
+            fns_aggregation = glob.glob(Path(self.root, "geoms", "aggregation_areas/*").as_posix())                
```

Comment:
I dont understand why you are reading the location of the geometries from this pre-defined path, while these should be saved in the spatial_joins obejct. You could just read the paths from there right?